### PR TITLE
Devcontainers & GHA Pipeline

### PR DESCRIPTION
**Pull request by pSchlarb**

### CODE REVIEW

File: .github/workflows/Push.yaml [hunk 1]
```diff
@@ -0,0 +1,138 @@
+name: libsovtoken Push
+
+on:
+  push:
+    branches:
+      - master
+      - Gitpod
```
Comment (WadeBarnes) [inline .github/workflows/Push.yaml:7]: Should be removed.

**Review by WadeBarnes — CHANGES_REQUESTED** [review]

File: .github/workflows/Push.yaml (hunk 1 above)
Comment (WadeBarnes) [reply .github/workflows/Push.yaml:7]: Remove GitPod

File: .github/workflows/Push.yaml [hunk 2]
```diff
@@ -0,0 +1,137 @@
+name: libsovtoken Push
+
+on:
+  push:
+    branches:
+      - stable
```
Comment (WadeBarnes) [inline .github/workflows/Push.yaml:6]: This should include `master` and `stable`.  Sorry if our conversation was not clear before.

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
Because we're planning on merging this into the stable branch we'll need to have the Jenkins build succeed as the gate.  The build is currently failing while trying to build a docker file

https://build.sovrin.org/blue/organizations/jenkins/libsovtoken%2Flibsovtoken-CI/detail/PR-447/3/pipeline/

image

**Review by WadeBarnes — APPROVED** [review]

File: .github/workflows/Push.yaml (hunk 2 above)
Comment (pSchlarb) [inline .github/workflows/Push.yaml:6]: Allow GHA to be run from Gitpod Branch
```suggestion
      - stable
      - Gitpod
```

**Review by WadeBarnes — APPROVED** [review]

**Pull request merged**